src/test/java/edu/isi/oba/ObaUtilsTest.java

```diff
@@ -78,8 +78,10 @@ public void missing_file () throws OWLOntologyCreationException{
 
     @Test
     public void run() {
-        String ontology1 = "https://mintproject.github.io/Mint-ModelCatalog-Ontology/release/1.4.0/ontology.xml";
-        String ontology2 = "https://knowledgecaptureanddiscovery.github.io/SoftwareDescriptionOntology/release/1.5.0/ontology.xml";
+        //See: https://github.com/mintproject/Mint-ModelCatalog-Ontology/tree/master/release/1.8.0/ontology.owl
+        String ontology1 = "https://raw.githubusercontent.com/mintproject/Mint-ModelCatalog-Ontology/master/release/1.8.0/ontology.owl";
+        //See: https://github.com/KnowledgeCaptureAndDiscovery/SoftwareDescriptionOntology/tree/master/release/1.9.0/ontology.owl
+        String ontology2 = "https://raw.githubusercontent.com/KnowledgeCaptureAndDiscovery/SoftwareDescriptionOntology/master/release/1.9.0/ontology.owl";
```

Comment:
The correct link should be https://knowledgecaptureanddiscovery.github.io/SoftwareDescriptionOntology/release/1.8.0/ontology.owl (I prefer that one to the raw.githubusercontent)

Reply:
Is there any reason for preferring the 1.8.0 version instead of the newest 1.9.0 one?

Reply:
no, you can use the 1.9.0 as well